.env

```diff
@@ -1,15 +0,0 @@
-REACT_APP_API_URL=https://us-central1-cv19assist-dev.cloudfunctions.net/api/v1
```

Comment:
Removed since `.env.staging` is used by default with `npm start` (which uses `react-scripts start`) as per [their docs](https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used)